examples/platform/silabs/FreeRTOSRuntimeStats.c

```diff
@@ -308,7 +308,6 @@ uint32_t ulGetAllTaskInfo(TaskInfo * taskInfoArray, uint32_t taskInfoArraySize,
     if (systemStats != NULL)
     {
         systemStats->terminatedTaskCount = deletedCount;
-        systemStats->totalTaskCount      = systemStats->activeTaskCount + deletedCount;
```

Comment:
This is not related to the energy modes tracing, was this intentional? I assume we compute it somewhere else for display but don't store it anymore?